pkg/test/ginkgo/options_monitor_events.go

```diff
@@ -114,12 +114,12 @@ func (o *MonitorEventsOptions) End(ctx context.Context, restConfig *rest.Config,
 	// this happens before calculation because events collected here could be used to drive later calculations
 	events, err = intervalcreation.InsertIntervalsFromCluster(ctx, restConfig, events, o.recordedResources, fromTime, endTime)
 	if err != nil {
-		return fmt.Errorf("InsertIntervalsFromClusterError: %w", err)
+		fmt.Fprintf(o.ErrOut, "InsertIntervalsFromCluster error but continuing processing: %w", err)
 	}
```

Comment:
I know the original code used `%w` but it will result in a weird output if that Fprintf were to run.  You can use `%v`.